addons/events/fnc_userKeyHandler.sqf

```diff
@@ -9,7 +9,7 @@ Author:
 ---------------------------------------------------------------------------- */
 #include "script_component.hpp"
 
-if (!GVAR(checkUserActions)) exitWith {};
+if (GVAR(skipCheckingUserActions)) exitWith {};
```

Comment:
lol

Reply:
@jonpas Check the commit info. Can't comment on commits anymore.

Reply:
Oh, wow.